test/jasmine/assets/mouse_event.js

```diff
@@ -14,7 +14,7 @@ module.exports = function(type, x, y, opts) {
         ev;
 
     if(type === 'scroll') {
-        ev = new window.WheelEvent('wheel', opts);
+        ev = new window.WheelEvent('wheel', Object.assign({}, fullOpts, opts));
```

Comment:
can you use `Lib.extendFlat` here instead - in case we run these tests on old browsers some day?